packages/atomic/src/components/common/layout/item-layout-controller.spec.ts

```diff
@@ -1,23 +1,25 @@
 import {LitElement} from 'lit';
 import {customElement} from 'lit/decorators.js';
 import {beforeEach, describe, expect, it, vi} from 'vitest';
-import * as displayOptions from './display-options';
+import type * as displayOptions from './display-options';
```

Comment:
If you apply my comment about renaming the module, you could import as displayUtils here for the sake of consistency.